src/ui/Filters/FilterOption.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Spring, animated } from 'react-spring';
+import Icon from 'ui/Icon/Icon';
+
+const Container = styled.div`
+  border-bottom: 0.5px solid ${props => props.theme.colors.terracota.main};
```

Comment:
For _all_ pixel values in here, please use the `rem()` helper method from `polished`